app/workers/notifications/schedule_reminder_mails_job.rb

```diff
@@ -0,0 +1,46 @@
+#-- encoding: UTF-8
+
+#-- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) 2012-2021 the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See docs/COPYRIGHT.rdoc for more details.
+#++
+
+module Notifications
+  class ScheduleReminderMailsJob < Cron::CronJob
+    # runs every hour, so 00:00, 01:00...
+    self.cron_expression = '0 * * * *'
+
+    def perform
+      User.having_reminder_mail_to_send(run_at).pluck(:id).each do |subscriber_ids|
```

Comment:
The plural for `subscriber_ids` does not make sense anymore after the latest changes. We could simply put user_id instead.
```suggestion
      User.having_reminder_mail_to_send(run_at).pluck(:id).each do |user_id|
```